src/modules/geocoder/geocoder-directive.js

```diff
@@ -69,10 +84,10 @@ angular.module('anol.geocoder')
                         scope.activeGeocoderConfig = initialConfigs[0]
                     }
 
-                    var removeMarkerInteraction;
-                    var translations = $translate([
```

Comment:
Altough i can not see where it is used, but Is this variable surely not needed anymore?